vignettes/function-flow.Rmd

```diff
@@ -121,38 +121,50 @@ input to extract the dates of all diabetes-specific podiatrist services.
 
 <!-- TODO: Add details on how this filtering should be done -->
 
+AAI: By date
```

Comment:
I'm guessing this is a note for yourself and that it will be elaborated on later 😋

Reply:
Yeah, it's a draft for a reason 😁